### PR TITLE
Sort-driver-source-with-minimum-distance-

### PR DESCRIPTION
**Pull request by gandhinikhil**
Sort available driver with driver current location with sources location.
Replace all Driver Model object with DriverDetailsVO.

### CODE REVIEW

File: src/main/java/com/example/cabbooking/vo/LocationDetailsVO.java [hunk 1]
```diff
@@ -2,11 +2,17 @@
...
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
+
 @AllArgsConstructor
 @NoArgsConstructor
 @Getter
+@Setter
 public class LocationDetailsVO {
     private int x;
```
Comment (sanskrati3) [inline src/main/java/com/example/cabbooking/vo/LocationDetailsVO.java]: please use proper variable naming convensions

Comment (gandhinikhil) [reply src/main/java/com/example/cabbooking/vo/LocationDetailsVO.java]: Changes variable name with proper naming

File: src/main/java/com/example/cabbooking/servicetest/DriverService.java [hunk 2]
```diff
@@ -1,8 +1,5 @@
 package com.example.cabbooking.servicetest;
```
Comment (sanskrati3) [inline src/main/java/com/example/cabbooking/servicetest/DriverService.java:2]: Note that the file names that you have provided should be like` DriverServiceTest  .`

Also the junits should follow https://www.baeldung.com/java-unit-testing-best-practices write more test cases both positive and negative cases like in case of exceptions also. 

Also please attach the screenshot in the description indicating how much coverage your test cases are covering , you can check this by running your` tests with coverage` option.

Comment (gandhinikhil) [reply src/main/java/com/example/cabbooking/servicetest/DriverService.java:2]: yes I will added more test case for both positive and negative.

File: src/main/java/com/example/cabbooking/controller/CabBookingController.java [hunk 3]
```diff
@@ -4,8 +4,9 @@
 import com.example.cabbooking.servicetest.RideService;
 import com.example.cabbooking.servicetest.UserService;
 import com.example.cabbooking.vo.DriverDetailsVO;
-import com.example.cabbooking.vo.LocationDetailsVO;
 import com.example.cabbooking.vo.RideDetailsVO;
+import com.example.cabbooking.vo.LocationDetailsVO;
+
 import com.example.cabbooking.vo.UserDetailsInVO;
 import lombok.RequiredArgsConstructor;
 import org.slf4j.Logger;
```
Comment (sanskrati3) [inline src/main/java/com/example/cabbooking/controller/CabBookingController.java:11]: always try to use deep import line no 17

Comment (gandhinikhil) [reply src/main/java/com/example/cabbooking/controller/CabBookingController.java:11]: yes will follow this now

**Comment by sanskrati3** [comment]
in your model files u dont need to manually write getter setter and to string constructors method , please use lombok annotations

**Review by sanskrati3 — APPROVED** [review]
LGTM

**Comment by gandhinikhil** [comment]
All required changes has done.
